tests/util/test_json.py

```diff
@@ -194,6 +195,23 @@ def as_dict(self):
     ) == {"$(BadData).bla": bad_data}
 
 
+def test_json_loads_array() -> None:
+    """Test json_loads_array validates result."""
+    assert json_loads_array('[{"c":1.2}]') == [{"c": 1.2}]
```

Comment:
Which json tool should be used for an array of objects?

Reply:
For now we only have helpers to check the top-level (`JsonValueType` for any json, or `JsonArrayType` or `JsonObjectType`).

We can add extra helpers if we want to ensure that each array item is an object but for now we don't have those.

Reply:
Why isn't this an array type then?